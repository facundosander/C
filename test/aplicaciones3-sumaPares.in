# crea un conjunto con rango 1..10
crearConjunto 10

# inserta 3 elementos
insertarConjunto 1
insertarConjunto 5
insertarConjunto 7
# imprime el conjunto
imprimirConjunto

# Chequea si hay un par que suma 6 (da true)
sumaPares 6

# Chequea si hay un par que suma 3 (da false)
sumaPares 3

# Chequea si hay un par que suma 8 (da true)
sumaPares 8

# Chequea si hay un par que suma 9 (da false)
sumaPares 9

# Chequea si hay un par que suma 12 (da true)
sumaPares 12

# inserta 2 elementos
insertarConjunto 2
insertarConjunto 4
# imprime el conjunto
imprimirConjunto

# Chequea si hay un par que suma 3 (da true)
sumaPares 3

# Chequea si hay un par que suma 7 (da true)
sumaPares 7

# Chequea si hay un par que suma 9 (da true)
sumaPares 9

# Chequea si hay un par que suma 1 (da false)
sumaPares 1

# Chequea si hay un par que suma 10 (da false)
sumaPares 10

# libera el conjunto
liberarConjunto

Fin
