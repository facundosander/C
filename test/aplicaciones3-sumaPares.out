1>#  crea un conjunto con rango 1..10.
2>3>#  inserta 3 elementos.
4>5>6>7>#  imprime el conjunto.
8>1 5 7 
9>#  Chequea si hay un par que suma 6 (da true).
10>Existe un par de elementos en el conjunto cuya suma es 6.
11>#  Chequea si hay un par que suma 3 (da false).
12>No existe un par de elementos en el conjunto cuya suma es 3.
13>#  Chequea si hay un par que suma 8 (da true).
14>Existe un par de elementos en el conjunto cuya suma es 8.
15>#  Chequea si hay un par que suma 9 (da false).
16>No existe un par de elementos en el conjunto cuya suma es 9.
17>#  Chequea si hay un par que suma 12 (da true).
18>Existe un par de elementos en el conjunto cuya suma es 12.
19>#  inserta 2 elementos.
20>21>22>#  imprime el conjunto.
23>1 2 4 5 7 
24>#  Chequea si hay un par que suma 3 (da true).
25>Existe un par de elementos en el conjunto cuya suma es 3.
26>#  Chequea si hay un par que suma 7 (da true).
27>Existe un par de elementos en el conjunto cuya suma es 7.
28>#  Chequea si hay un par que suma 9 (da true).
29>Existe un par de elementos en el conjunto cuya suma es 9.
30>#  Chequea si hay un par que suma 1 (da false).
31>No existe un par de elementos en el conjunto cuya suma es 1.
32>#  Chequea si hay un par que suma 10 (da false).
33>No existe un par de elementos en el conjunto cuya suma es 10.
34>#  libera el conjunto.
35>36>Fin.
